spec/components/chat_preview_spec.rb

```diff
@@ -4,6 +4,9 @@
 
 RSpec.describe ChatPreview::ChatPreview, type: :component do
   subject { render_inline(described_class.new(**params)) }
+  before do
```

Comment:
I haven’t really understood why removing the respective environment variables from the setup is necessary or why you prefer stubbing the settings in all those specs?

Reply:
That was a refactoring for code quality: it's considered bad practice to write into `ENV` and we do this for our test environment. Second, you have to look into several locations to understand the test, the assertion is in the spec and the stubbed value in `config/environments/test.rb`.

Besides that I wanted to make sure that we never commit our Threema API keys by adding:
```
  c.filter_sensitive_data('THREEMARB_API_SECRET') { ENV['THREEMARB_API_SECRET'] }
  c.filter_sensitive_data('THREEMARB_PRIVATE') { ENV['THREEMARB_PRIVATE'] }
  ```
which reads from `ENV`.